### PR TITLE
chihiro: EMA decay sweep {0.9995, 0.9997} vs SOTA 0.999

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

The current SOTA stack uses `--ema-decay 0.999` (10× faster than the typical EMA convention of 0.9999). At our 11k-step training budget, decay=0.999 has an effective averaging window of ~1000 steps (~1 epoch). **Hypothesis: a longer EMA window — decay ∈ {0.9995, 0.9997} — provides better variance reduction late in training where the cosine LR schedule has already settled, and may particularly help slow-converging tau_y/tau_z channels by reducing per-step noise in their gradient updates.**

The EMA window size at decay d for w training steps is approximately 1/(1-d):
- 0.999 → 1000 steps (~1 epoch) — current
- 0.9995 → 2000 steps (~2 epochs)
- 0.9997 → 3333 steps (~3 epochs)
- 0.9999 → 10000 steps (~10 epochs, full training)

Too-long EMA windows hurt early training (averaging stale random-init weights), but the SOTA stack uses `--ema-start-step` to skip warmup. With proper warmup, longer decay should be safe and beneficial.

## Instructions

This is a **2-arm sweep** comparing EMA decay {0.9995, 0.9997} against the SOTA decay 0.999.

1. Run two trainings on the **canonical SOTA stack** with only `--ema-decay` changed:
   - Arm A: `--ema-decay 0.9995 --wandb-group chihiro-ema-decay-9995`
   - Arm B: `--ema-decay 0.9997 --wandb-group chihiro-ema-decay-9997`

2. **Reproduce command for Arm A** (replace decay for Arm B):

```bash
torchrun --standalone --nproc_per_node=8 train.py \
  --agent chihiro --optimizer lion --lr 1e-4 --weight-decay 5e-4 \
  --no-compile-model --batch-size 4 --validation-every 1 \
  --train-surface-points 65536 --eval-surface-points 65536 \
  --train-volume-points 65536 --eval-volume-points 65536 \
  --vol-points-schedule "0:16384:3:32768:6:49152:9:65536" \
  --model-layers 4 --model-hidden-dim 512 --model-heads 4 --model-slices 128 \
  --ema-decay 0.9995 --grad-clip-norm 0.5 --lr-warmup-epochs 1 \
  --pos-encoding-mode string_separable --use-qk-norm \
  --rff-num-features 16 --rff-init-sigmas "0.25,0.5,1.0,2.0,4.0" \
  --lr-cosine-t-max 13 --epochs 13 \
  --use-gradnorm --gradnorm-mode ema_proxy \
  --gradnorm-alpha 0.5 --gradnorm-ema-beta 0.9 --gradnorm-min-weight 0.7 \
  --wandb-group chihiro-ema-decay-9995
```

3. **Run BOTH arms back-to-back** on your single 8-GPU pod (each arm is ~4.7 hours; budget 360 minutes per arm => 75% timeout = 270min effective per arm, fits comfortably).

4. **Critical**: each arm must complete the full 13 epochs (or hit the 270min timeout, whichever first). Report per-arm:
   - Per-epoch val_abupt, val_tau_y, val_tau_z
   - Best-val checkpoint test metrics (test_abupt, test_tau_y, test_tau_z)
   - W&B run IDs (rank-0)

5. Use `--wandb-group` exactly as specified so the runs group correctly.

## Baseline

Current SOTA (PR #523 thorfinn EMA-proxy GradNorm, ema-decay=0.999, EP6, run `wyz68o8r`):
- val_abupt = **6.9246%**
- test_abupt = **8.2355%**
- val tau_y = 8.7197%, val tau_z = 10.2960%
- test tau_y = 8.4656%, test tau_z = 9.6720%

**Win condition: at least one arm has val_abupt < 6.9246% AND test_abupt < 8.2355%.** Both arms beating baseline = a clear EMA-decay direction win, route to follow-up sweep.

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #558 because ADVISOR retracting: misrouted assignment. The student does not have a deployed pod on the tay (drivaerml-ddp8) branch. Reopening on a branch with the student's deployment if relevant..
